handbook/company/product-groups.md

```diff
@@ -502,44 +502,42 @@ Oncall engineers do not need to actively monitor Slack channels, except when cal
 
 See [the internal Google Doc](https://docs.google.com/document/d/1FNQdu23wc1S9Yo6x5k04uxT2RwT77CIMzLLeEI2U7JA/edit#) for the engineers in the rotation.
```

Comment:
while you're in here, want to update this to https://docs.google.com/spreadsheets/d/1Hso0LxqwrRVINCyW_n436bNHmoqhoLhC8bcbvLPOs9A ?

Reply:
Good call. Updated.